### PR TITLE
feat(zkevm): introduction to zkevm 

### PR DESCRIPTION
**Pull request by EmpieichO**
Adding an md-file to Introduction to the Polygon zkEVM Wiki, plus modification to sidebar.js

### CODE REVIEW

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  - bridge
+  - zkEVM
+  - Polygon zkEVM
+image: https://wiki.polygon.technology/img/thumbnail/polygon-zkevm.png
+
+---
+
+
+
+# Polygon zkEVM Documentation
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:24]: I'd reconsider the header namings on the page:
- "Polygon zkEVM Documentation" is implied by the content already. A simple "Introduction to zkEVM" or "Overview" would work better.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+# Polygon zkEVM Documentation 
+
+
+
+Welcome to Polygon zkEVM documentation.
+
+Polygon zkEVM is a decentralized Ethereum Layer 2 scalability solution based on cryptographic proofs to provide validation and fast finality to transactions.
+
+This documentation presents
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:32]:
```suggestion
> Polygon Hermez, the predecessor of zkEVM,  was launched in March 2021. Its goal was to scale payments
> and transfers of ERC-20 tokens. It currently achieves up to 2000 transactions per second (TPS). See
> [Ethereum Live TPS](https://ethtps.info/Network/Ethereum) to track Ethereum's TPS.

The zkEVM carries out state changes, which come from executions of Ethereum's Layer 2 transactions that
users send to the network and subsequently produces validity proofs attesting to the correctness of the
state change computations carried out off-chain.

> Although taking on this revolutionary design approach was hard, the objective is to minimize the users and
> dApps friction when using the solution, which requires the recreation of all EVM opcodes for the transparent deployment of existing Ethereum smart contracts. As a result, the team has created and engineered a new set of technologies and tools.

This documentation presents: 
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+
+
+Welcome to Polygon zkEVM documentation.
+
+Polygon zkEVM is a decentralized Ethereum Layer 2 scalability solution based on cryptographic proofs to provide validation and fast finality to transactions.
+
+This documentation presents 
+
+- What Polygon zkEVM is
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:34]: Would suggest reorganizing the list to showcase a better flow, that is, starting with "What is" -> "How tos" -> "Participate/Get Involved"
```
- What Polygon zkEVM is
- The architecture of Polygon zkEVM (I wouldn't limit this to an overview as there is solid material at describing the various features and components:) )
- zkEVM's design approach to state machines
- How to run a local zkNode
- How to participate in the zkEVM testnet, and ultimately build on the mainnet

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+- How to participate in the Polygon zkEVM testnet, and ultimately building on the mainnet
+- The architectural overview of Polygon zkEVM
+- How to run a local zkNode and a production zkNode
+- The Polygon zkEVM's Design approach to state machines.
+
+The document then cascades step-by-step into the finer details of zkEVM's sub-components, such as the zero-knowledge Prover (zkProver), the state machines used in the zkProver, the Layer X to Layer Y bridge smart contract, as well as the special toolings enabling the Polygon zkEVM achieve its objectives.
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:40]:
```suggestion
The documentation cascades into the finer details of zkEVM's sub-components, including the zero-knowledge Prover (zkProver), the state machines used in the zkProver, the Layer X <> Layer Y bridge smart contract, and the special toolings that enable the zkEVM to achieve its objectives. 
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+- What Polygon zkEVM is
+- How to participate in the Polygon zkEVM testnet, and ultimately building on the mainnet
+- The architectural overview of Polygon zkEVM
+- How to run a local zkNode and a production zkNode
+- The Polygon zkEVM's Design approach to state machines.
+
+The document then cascades step-by-step into the finer details of zkEVM's sub-components, such as the zero-knowledge Prover (zkProver), the state machines used in the zkProver, the Layer X to Layer Y bridge smart contract, as well as the special toolings enabling the Polygon zkEVM achieve its objectives. 
+
+We however emphasise that this documentation is still **Work In Progress** (WIP). Some aspects have been covered in more detail, while others still need further expansion. Also, sections of this documentation might later be reorganized in order to achieve better flow.
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:42]: I'd put this as an alert at the top

```
:::caution Documentation status
This documentation is still **Work In Progress** (WIP). Some aspects have been covered in more detail, while others still need further expansion. Also, sections of this documentation might later be reorganized in order to achieve a better flow.
:::
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+
+
+## About Polygon zkEVM
+
+
+
+Polygon zkEVM is a powerful, decentralized technology developed to provide Layer 2 scalability solutions to Ethereum blockchain users. It is therefore designed to seamlessly integrate with the Ethereum ecosystem. Given the current and anticipated increase in the number of Ethereum's on-chain transactions, the Layer 1 solution is already facing the blockchain trilemma: Balancing between decentralization, scalability, and security. This is where Polygon zkEVM steps in. By providing zero-knowledge-rollups (zk-rollups) that sit on top of the Ethereum Mainnet, the scalability and the transactions per second (TPS) can be dramatically improved.
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:50]: The content under "About Polygon zkEVM" and "What is Polygon zkEVM" are redundant and can be compiled into one section,

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+
+
+# Polygon zkEVM Documentation 
+
+
+
+Welcome to Polygon zkEVM documentation.
+
+Polygon zkEVM is a decentralized Ethereum Layer 2 scalability solution based on cryptographic proofs to provide validation and fast finality to transactions.
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:30]:
```suggestion
Polygon zkEVM is a decentralized Ethereum Layer 2 scalability solution based on cryptographic proofs to provide validation and fast finality to transactions. By providing zero-knowledge rollups (zk-rollups) that sit on top Ethereum, the scalability and the transactions per second (TPS) can be dramatically improved.
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 6 above)
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:50]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:31]:
```suggestion
zkEVM utilizes cryptographic zero-knowledge technology to provide validation and fast finality of off-chain transactions. It has been designed to emulate the Ethereum Virtual Machine (EVM) by recreating all existing EVM opcodes.

```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+
+## About Polygon zkEVM
+
+
+
+Polygon zkEVM is a powerful, decentralized technology developed to provide Layer 2 scalability solutions to Ethereum blockchain users. It is therefore designed to seamlessly integrate with the Ethereum ecosystem. Given the current and anticipated increase in the number of Ethereum's on-chain transactions, the Layer 1 solution is already facing the blockchain trilemma: Balancing between decentralization, scalability, and security. This is where Polygon zkEVM steps in. By providing zero-knowledge-rollups (zk-rollups) that sit on top of the Ethereum Mainnet, the scalability and the transactions per second (TPS) can be dramatically improved.
+
+In order to prove that the off-chain computations are correct, Polygon zkEVM employs easily verifiable zero-knowledge proofs. The Layer 2 zero-knowledge proofs are based on the complex polynomial computations that provide fast finality to off-chain transactions.
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:52]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+
+
+
+## What is Polygon zkEVM?
+
+
+
+Polygon zkEVM, henceforth zkEVM, is a decentralized Ethereum Layer 2 scalability solution utilising cryptographic zero-knowledge technology in order to provide validation and fast finality of off-chain transaction computations. It has been designed and developed to emulate the Ethereum Virtual Machine (EVM) by recreating all existing EVM opcodes for transparent deployment of existing Ethereum smart contracts.
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:60]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+
+
+
+Polygon zkEVM is a powerful, decentralized technology developed to provide Layer 2 scalability solutions to Ethereum blockchain users. It is therefore designed to seamlessly integrate with the Ethereum ecosystem. Given the current and anticipated increase in the number of Ethereum's on-chain transactions, the Layer 1 solution is already facing the blockchain trilemma: Balancing between decentralization, scalability, and security. This is where Polygon zkEVM steps in. By providing zero-knowledge-rollups (zk-rollups) that sit on top of the Ethereum Mainnet, the scalability and the transactions per second (TPS) can be dramatically improved.
+
+In order to prove that the off-chain computations are correct, Polygon zkEVM employs easily verifiable zero-knowledge proofs. The Layer 2 zero-knowledge proofs are based on the complex polynomial computations that provide fast finality to off-chain transactions.
+
+
+
+## What is Polygon zkEVM?
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:56]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+- How to run a local zkNode and a production zkNode
+- The Polygon zkEVM's Design approach to state machines.
+
+The document then cascades step-by-step into the finer details of zkEVM's sub-components, such as the zero-knowledge Prover (zkProver), the state machines used in the zkProver, the Layer X to Layer Y bridge smart contract, as well as the special toolings enabling the Polygon zkEVM achieve its objectives. 
+
+We however emphasise that this documentation is still **Work In Progress** (WIP). Some aspects have been covered in more detail, while others still need further expansion. Also, sections of this documentation might later be reorganized in order to achieve better flow.
+
+
+
+## About Polygon zkEVM
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:46]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 3 above)
Comment (DannyS03) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:34]: I'd also link to a "general page" for each of these points to establish the list as an index and a way users can get started.

**Review by DannyS03 — CHANGES_REQUESTED** [review]
thank you @EmpieichO, can we also please remove the extra whitespaces on the page, and format the metadata so that "id" starts at line 2.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 2 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:32]: Suggested change not clear

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 1 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:24]: Consider SEO, users are most likely to search for "Polygon zkEVM Documentation" and the Wiki will ultimately be the only place for documentation.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 4 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:40]: Done.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 5 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:42]: Done.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 6 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:50]: Done. One section now.

Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:50]: I can't spot the difference here.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 1 above)
Comment (ashucoder9) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:24]: Hey @EmpieichO, as per SEO requirements we already have the title element as "Polygon zkEVM Documentation". If you search even now, you will find that Polygon Wiki's Architecture of zkEVM appears second on Google, which was originally crawled as the main page for zkEVM docs on Polygon Wiki. Later, we changed that to "Build on zkEVM".

As we plan to deprecate the original Hermez docs website, search results will point directly to Polygon Wiki's zkEVM article. As per Danny's suggestion, the title "Introduction to zkEVM" fits better.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
+---
+
+id: intro-polygon-zkevm
+title: Polygon zkEVM Documentation
+sidebar_label: Polygon zkEVM Documentation
+description: Introduction to the Polygon zkEVM Documentation
+keywords:
+
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:8]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+  - zk rollups
+  - polygon
+  - proof
+  - efficiency
+  - hermez
+  - bridge
+  - zkEVM
+  - Polygon zkEVM
+image: https://wiki.polygon.technology/img/thumbnail/polygon-zkevm.png
+
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:19]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+---
+
+
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:22]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 16]
```diff
@@ -0,0 +1,77 @@
...
+  - proof
+  - efficiency
+  - hermez
+  - bridge
+  - zkEVM
+  - Polygon zkEVM
+image: https://wiki.polygon.technology/img/thumbnail/polygon-zkevm.png
+
+---
+
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:21]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 17]
```diff
@@ -0,0 +1,77 @@
...
+The document then cascades step-by-step into the finer details of zkEVM's sub-components, such as the zero-knowledge Prover (zkProver), the state machines used in the zkProver, the Layer X to Layer Y bridge smart contract, as well as the special toolings enabling the Polygon zkEVM achieve its objectives. 
+
+We however emphasise that this documentation is still **Work In Progress** (WIP). Some aspects have been covered in more detail, while others still need further expansion. Also, sections of this documentation might later be reorganized in order to achieve better flow.
+
+
+
+## About Polygon zkEVM
+
+
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:48]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:49]:
```suggestion
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 6 above)
Comment (ashucoder9) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:50]: That's actually a suggestion to delete the mentioned lines. You can simply click on "Commit Suggestion" for suggestions above to commit them into this doc.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 19]
```diff
@@ -0,0 +1,77 @@
...
+## What is Polygon zkEVM?
+
+
+
+Polygon zkEVM, henceforth zkEVM, is a decentralized Ethereum Layer 2 scalability solution utilising cryptographic zero-knowledge technology in order to provide validation and fast finality of off-chain transaction computations. It has been designed and developed to emulate the Ethereum Virtual Machine (EVM) by recreating all existing EVM opcodes for transparent deployment of existing Ethereum smart contracts.
+
+The predecessor, Polygon Hermez 1.0, was launched in March 2021. Its goal was to scale payments and transfers of ERC-20 tokens. It currently achieves up to 2000 transactions per second (tps), and thus accomplished the purpose it was designed for. See [Ethereum Live TPS](https://ethtps.info/Network/Ethereum) to keep track of Ethereum's live transactions per second.
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:62]:
```suggestion
The predecessor, Polygon Hermez 1.0, was launched in March 2021. Its goal was to scale payments and transfers of ERC-20 tokens. It currently achieves up to 2000 transactions per second (TPS), and thus accomplished the purpose it was designed for. See [Ethereum Live TPS](https://ethtps.info/Network/Ethereum) to keep track of Ethereum's live transactions per second.
```

Comment (ashucoder9) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:62]: This is a bit redundant as the content here already appears in the first paragraph after suggestions are merged.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 20]
```diff
@@ -0,0 +1,77 @@
...
+
+The zkEVM carries out state changes, which come from executions of Ethereum’s Layer 2 transactions that users send to the network, and subsequently produces validity proofs attesting to the correctness of the state change computations carried out off-chain.
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:64]:
```suggestion
The zkEVM carries out state changes, which come from executions of Ethereum’s Layer 2 transactions that users send to the network and subsequently produces validity proofs attesting to the correctness of the state change computations carried out off-chain.
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 21]
```diff
@@ -0,0 +1,77 @@
...
+
+Although taking on this revolutionary design approach was a hard decision to make, the objective is to minimise the users and dApps friction when using the solution. It is an approach that requires recreation of all EVM opcodes for the transparent deployment of existing Ethereum smart contracts. For this purpose, a new set of technologies and tools has been created and engineered by the team.
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:66]:
```suggestion
Although taking on this revolutionary design approach was a hard decision to make, the objective is to minimize the users and dApps friction when using the solution. It is an approach that requires the recreation of all EVM opcodes for the transparent deployment of existing Ethereum smart contracts. For this purpose, a new set of technologies and tools has been created and engineered by the team.
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 8 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:31]: Not sure how to resolve this one. The idea is to start with a more general Ethereum context before coming to the specific way zkEVM provides validation.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 22]
```diff
@@ -0,0 +1,77 @@
...
+
+
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:68]: Please remove these blank lines too @EmpieichO

**Review by ashucoder9 — CHANGES_REQUESTED** [review]
Please check the suggestions and commit them into your PR to update this document.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 10 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:60]: See the new version.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 11 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:56]: I wish to retain this heading, see new version.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 3 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:34]: Done.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 7 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:30]: see new version first

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 8 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:31]: see new version

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 22 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:64]: Done

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 23]
```diff
@@ -0,0 +1,61 @@
...
+- The architectural overview of Polygon zkEVM
+- The Polygon zkEVM's Design approach to state machines
+- How to run a local zkNode
+- How to participate in the Polygon zkEVM testnet and ultimately build on the mainnet.
+
+The documentation cascades into the finer details of zkEVM's sub-components, including the zero-knowledge Prover (zkProver), the state machines used in the zkProver, the Layer X <> Layer Y bridge smart contract, and the special toolings that enable the zkEVM to achieve its objectives.
+
+
+
+```
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:41]: Please remove these backticks. They are not appropriate here. We only need ::: for admonitions.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 24]
```diff
@@ -0,0 +1,61 @@
...
+::: Caution on Documentation status
+This documentation is still **Work In Progress** (WIP). Some aspects have been covered in more detail, while others still need further expansion. Also, sections of this documentation might later be reorganized in order to achieve a better flow.
+:::
+```
```
Comment (ashucoder9) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:45]: Remove this too @EmpieichO ser!

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 23 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md]: removed

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md (hunk 24 above)
Comment (EmpieichO) [reply docs/zkEVM/Intro-Polygon-zkEVM-Docs.md]: okay

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 25]
```diff
@@ -0,0 +1,52 @@
...
+
+caution  Documentation status
+This documentation is still **Work In Progress** (WIP). Some aspects have been covered in more detail, while others still need further expansion. Also, sections of this documentation might later be reorganized in order to achieve a better flow.
+
+
+## What is Polygon zkEVM?
+
+Polygon zkEVM is a powerful, decentralized technology developed to provide Layer 2 scalability solutions to Ethereum blockchain users. It is therefore designed to seamlessly integrate with the Ethereum ecosystem. Given the current and anticipated increase in the number of Ethereum's on-chain transactions, the Layer 1 solution is already facing the blockchain trilemma: Balancing between decentralization, scalability, and security. This is where Polygon zkEVM steps in. By providing zero-knowledge-rollups (zk-rollups) that sit on top of the Ethereum Mainnet, the scalability and the transactions per second (TPS) can be dramatically improved.
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:44]:
```suggestion
Polygon zkEVM is a powerful, decentralized Layer 2 scalability solution for Ethereum. Given that Ethereum is subject to the DLT (distributed ledger technology) trilemma, it cannot scale beyond its transaction threshold without sacrificing decentralization or security. 

By providing zero-knowledge rollups (ZK rollups) that sit on top of the Ethereum Mainnet, the scalability and the transactions per second (TPS) can be dramatically improved.
```

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 26]
```diff
@@ -0,0 +1,52 @@
...
+
+In order to prove that the off-chain computations are correct, Polygon zkEVM employs verifiable zero-knowledge proofs as validity proofs. Although the Layer 2 zero-knowledge proofs are based on complex polynomial computations to provide validation and finality to off-chain transactions, the validity proofs are quick and easy to verify.
+
+Polygon zkEVM, henceforth zkEVM, is a virtual machine, designed and developed to emulate the Ethereum Virtual Machine (EVM) by recreating all existing EVM opcodes for transparent deployment of existing Ethereum smart contracts.
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:48]: This should be the first statement in this section, as it introduces zkEVM for what it is, and then the section can go into capabilities.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 27]
```diff
@@ -0,0 +1,52 @@
...
+
+As a state machine, zkEVM carries out state changes, which come from executions of Ethereum’s Layer 2 transactions that users send to the network, and subsequently produces validity proofs attesting to the correctness of the state change computations carried out off-chain.
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:50]: This and line 46 can be combined into one.

File: docs/zkEVM/Intro-Polygon-zkEVM-Docs.md [hunk 28]
```diff
@@ -0,0 +1,52 @@
...
+  - zk rollups
+  - polygon
+  - proof
+  - efficiency
+  - hermez
+  - bridge
+  - zkEVM
+  - Polygon zkEVM
+image: https://wiki.polygon.technology/img/thumbnail/polygon-zkevm.png
+
```
Comment (DannyS03) [inline docs/zkEVM/Intro-Polygon-zkEVM-Docs.md:19]:
```suggestion
```

**Comment by ashucoder9** [comment]
I'm closing this PR as discussed with @EmpieichO

Will raise another PR for the same article with the required modifications and content guidelines as recommended for Polygon Wiki.

**Pull request closed without merging**